internal/sirius/payment.go

```diff
@@ -0,0 +1,36 @@
+package sirius
+
+import "fmt"
+
+type Payment struct {
+	ID          int           `json:"id,omitempty"`
+	CaseID      int           `json:"case_id,omitempty"`
+	Source      PaymentSource `json:"source"`
```

Comment:
I'm changing this to just be the code string (e.g. `PHONE`, `MAKE`), since the `Name` part is an implementation detail that we shouldn't be exposing.